doc/OnlineDocs/contributed_packages/pyros.rst

```diff
@@ -448,34 +460,33 @@ We can then loop through this array and call PyROS within a loop to identify rob
 
 .. code::
 
+  >>> # This takes a long time to run and therefore is not a doctest
   >>> # === An array of maximum relative deviations from the nominal uncertain parameter values to utilize in constructing box sets
   >>> relative_deviation_list = [0.00, 0.10, 0.20, 0.30, 0.40]
-
   >>> # === Final robust optimal objectives
   >>> robust_optimal_objectives = []
-
-  >>> for relative_deviation in relative_deviation_list:
-      ... bounds = [(nominal_values[i] - relative_deviation*nominal_values[i],
-                     nominal_values[i] + relative_deviation*nominal_values[i])
-                     for i in range(4)]
-      ... box_uncertainty_set = pyros.BoxSet(bounds = bounds)
-      ... results = pyros_solver.solve(model = m,
-                                       first_stage_variables = first_stage_variables,
-                                       second_stage_variables = second_stage_variables,
-                                       uncertain_params = uncertain_parameters,
-                                       uncertainty_set = box_uncertainty_set,
-                                       local_solver = local_solver,
-                                       global_solver = global_solver,
-                                       options = {
-                                          "objective_focus": pyros.ObjectiveType.worst_case,
-                                          "solve_master_globally": True,
-                                          "decision_rule_order": 1
-                                       })
-      ... if results.grcs_termination_condition != pyros.grcsTerminationCondition.robust_optimal:
-      ...   print("This instance didn't solve to robust optimality.")
-      ...   robust_optimal_objective.append("-----")
-      ... else:
-      ...   robust_optimal_objectives.append(str(results.final_objective_value))
+  >>> for relative_deviation in relative_deviation_list: # doctest: +SKIP
```

Comment:
Why is this doctest being skipped?

Reply:
This doctest takes a very long time to run. And since it is solving the previous two doctests as part of the loop, I think testing the first two cases is sufficient here.